### ISSUE TITLE
The other 10%

### ISSUE THREAD

**Issue by awhedbee22** [cyrixmorten/parse-angular-crud-demo#1]
Thanks for this upload. I find it very helpful. 

Couple questions. 

When i create a new Guard ID with Name, It only displays the ID but not the name (The name shows in my parse dashboard). Also when I update the new Guard, only the ID updates, and the name does not. Is this just the functionality that you have so far?

I see that you were able to Create Read with that snippet in the readme. Care to share the rest?

Thank you.

**Comment by cyrixmorten** [comment]
Hi glad you like it

Thank you for reporting this issue.

This demo is directly spawned from a project that I am currently working on and had simply missed some variable name updates at the last commit. It should be working as expected now.

I will at some point, hopefully next week, update the readme with a walkthrough of adding your own parse object to make it clear where the important parts of the code are.

Just to give a quick sum-up here.
Suppose you would like to add a Cars object, registering the names of cars, then:

1) Locate `js/app.js` and add `when('/cars', 'cars')` and `segment('cars', {templateUrl: 'partials/cars.html', controller: 'CRUDCtrl'})` 

2) In index.html  add an entry to the navbar `<li ng-class="{active: $routeSegment.startsWith('cars')}"><a href="#/cars">CarsParseObject</a></li>`

3) In `js/services.js` add the definition of the Cars object:

```
app.factory('ParseCarsObject', [ 'ParseService', function(ParseService) {

    var factory = {};

    factory.getParseObject = new ParseService.StandardParseObject({
        objectname : 'Cars',
        attrs : [ 'owner', 'name' ]
    }, {
        name : ''
    }, function(car) {
        return {
            name : guard.getName()
        }
    }, {
        ACL : new Parse.ACL(ParseService.Account.getCurrentUser()),
        owner : ParseService.Account.getCurrentUser()
    });

    return factory;

} ]);
```

4) Use the definition to add a new Controller in `js/controllers.js`

```
controllerModule.controller('CarsCtrl', [ '$scope', 'ParseCarsObject',
        function($scope, ParseCarsObject) {

            $scope.$parent.ParseCrud = ParseCarsObject.getParseObject;

        } ]);
```

5) And finally create a view in partials `partials/cars.html` to display the table, in this case I just use a copy of `circuits.html` as both objects are essentially equal, only using a name entry. Notice the `ng-controller="CarsCtrl"`, which is important to point the view to the right controller.

```
<div class="table-scrollable" ng-controller="CarsCtrl">
    <div class="panel panel-default">
        <div class="panel-heading">
            <h3 class="panel-title">Create circuit</h3>
        </div>
        <div class="panel-body ">
            <table class="table">
                <tr>

                    <td><input placeholder="Name" class="form-control"
                        type="text" ng-model="createData.name" /></td>

                    <td data-title="''" width="200">
                        <a href="" class="btn btn-default" ng-click="add()">Create</a>
                    </td>
                </tr>
            </table>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <h3 class="panel-title">Circuits</h3>
        </div>
        <div class="panel-body">
            <table ng-table="tableParams" class="table">
                <tr ng-repeat="crudObject in crudObjects">
                    <td data-title="'Name'"><input ng-readonly="!crudObject.$edit"
                        class="form-control" type="text" ng-model="crudObject.name" />
                    </td>
                    <td data-title="''" width="200"><crud-edit-delete /></td>
                </tr>
            </table>
        </div>
    </div>
</div>
```

**Comment by awhedbee22** [comment]
Thank you!

That is all very helpful! The creator of the patch did not do a great job at documenting it..

I appreciate it and keep up the good work!

**Comment by cyrixmorten** [comment]
You are welcome :)

Yeah, I had a hard time figuring it out myself, and when I finally figured a way to overcome the problem with not having two-way data binding (posted an issue about it on the patch site), I went ahead and made this demo.

I also found his demo hard to understand, though they should be working on applying the patch to it, hopefully making it easier to extract useful parts of the code to other projects. Do not know what they do about the data binding issue though..

Have made a small typo correction in the 1) bullet of the small guide, hope I didn't miss more of them, otherwise don't hesitate to get back to me.

Happy coding
